src/pages/learn/index.tsx

```diff
@@ -0,0 +1,943 @@
+                isExternal
+              >
+                Decentralizing Everything
+              </DocLink>
+            </DocsContainer>
+          </Section>
+
+          <Section>
+            <h2 id={tocItems[1].id}>{tocItems[1].title}</h2>
+            <p>
+              It’s actually quite simple once you get the hang of it. You need
+              an application that is commonly called a “wallet”. It's an app
+              that helps you store your funds and to authenticate and interact
+              with applications on Ethereum.
+            </p>
+            <CardGrid>
+              <Card
+                title="What is a wallet?"
+                description="How you manage your ETH and your Ethereum accounts."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.wallet)}
+                      alt="Illustration of a robot."
+                    />
+                  </CardImage>
+                  <ButtonLink to="/wallets/">What is a wallet?</ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Find a wallet"
+                description="We will help you find the right wallet based on the features you care about."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.futureTransparent)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/wallets/find-wallet/">
+                    List of wallets
+                  </ButtonLink>
+                </>
+              </Card>
+              <Card
+                title="Crypto security 101"
+                description="Learn how to identify scams and how to avoid the most common tricks."
+              >
+                <>
+                  <CardImage>
+                    <GatsbyImage
+                      image={getImage(data.dogeComputer)}
+                      alt=""
+                    />
+                  </CardImage>
+                  <ButtonLink to="/security/">Crypto security 101</ButtonLink>
+                </>
+              </Card>
+            </CardGrid>
+
+            <Banner>
+              <BannerBody>
+                <h3>Things to consider when using Ethereum</h3>
+                <ul>
+                  <li>
+                    Each Ethereum transaction requires a fee in the form of ETH,
+                    even if you need to move different tokens built on Ethereum
+                    like the stablecoins USDC or DAI.
+                  </li>
+                  <li>
+                    Fees can be high depending on the demand so we recommend
+                    using <Link to="/layer-2/">Layer 2s</Link>.
+                  </li>
+                  <li>
+                    There are options for custodial and non-custodial wallets.
```

Comment:
Can we put an alternative here maybe @konopkja? We're introducing a concept that assumes they've read (and understood) the wallets page. I'd at very least reword this to be jargonless if you feel strongly about keeping the point itself.

Reply:
@minimalsm  i would keep layer 2s in there, but i think "There are options for custodial and non-custodial wallets." can be removed